### ISSUE TITLE
create of new window is not possible (using key shortcut)

### ISSUE THREAD

**Issue by taraszka** [Microsoft/WSL#1183]
Hi,
- A brief description
  I am not able to open new window of bash on ubuntu on windows 10 using any keyboard shourtcut that I know either using newly created keyboard combination under application shortcut on my desktop.
  Each time I ran a combination of my keyboard shortcut (like in my example ctrl + alt + N) it back to firstly created window by this shortcut instead of opening new window/process.
- Expected results
  I am expecting to have a new bash on ubuntu on windows window opened.
- Actual results (with terminal output if applicable)
  NA
- Your Windows build number
  This is my Windows 10 version: Update for Windows 10 Version 1607 for x64-based Systems (KB3176936)
- Steps / All commands required to reproduce the error from a brand new installation 
  Step 1. The first step is to create desktop shortcut for the bash on ubuntu on windows using keyboard shortcut. To do so, simply open the Start menu, and drag-and-drop app from the Start menu to the desktop to create desktop shortcuts for the selected app.
  Step 2.  Once the app shortcut is on desktop, right-click on it, and then click Properties.
  Step 3. Under the Shortcut tab, select the field next to Shortcut key.
  Step 4. Press a key that you want to use to launch the app. In my case i choosen ctrl-alt-N
  Step 5. Once done, click Apply button.
  Step 6. Press Ctrl-Alt-N, new bash window should apear.
  Step 7. Press again ctrl-alt-N, no new window aprear. If you would click and run another instance of bash on ubuntu on windows the new window will apear but if you'd like to press again ctrl-alt-N instead of new window the old one will pop up.
- Strace of the failing command
  NA
- Required packages and commands to install
  NA

**Comment by Manouchehri** [comment]
Have you tried ConEmu?

**Comment by fpqc** [comment]
I didn't even know you could set shortcuts like that.

**Comment by zadjii-msft** [comment]
Does this work with other applications? For example, if you bind Ctrl+Alt+E to launch File Explorer, can you use it to launch multiple instances of explorer? Because that doesn't work for me. I'd imagine this is a windows shell bug if it's supposed to work.

For the record, I never know about this mechanism either, though it's very handy. I've personally used the win key+number shortcut to launch applications. For ex, if you pin Bash on Windows to the taskbar, and move it to the "first" position (right after cortana and the task view button for me), then `Win Key + 1` will launch a bash window if one doesn't exist, or bring it to the foreground. `Win Key+Shift+1` will launch a new window always.

**Comment by jbarchuk** [comment]
I found this thread because GH is the first ummm... 'app,' (we used to call them 'programs' LOL!) that I've ever seen that can't use a ctrl-alt-button Keyboard Shortcut combination. The option simply does not appear in in icon -> right click -> properties. The other happened only a few years ago, Autodesk Fusion360. My experience with Win goes back to 1.1 and KS was one of the must useful features that I have **ALWAYS** used!! I mean always-always. I *HATE*HATE*HATE* blanking my screen down to the raw desktop just to turn on a program. And yeah it's FILLED with icons because one of the requirements of KS is that it's applied and managed -via- the desktop icon.

So when it failed the first thing I did was hunt for the F360 executable. The common thread I see between GH and F360 is that the actual software does not reside in the usual c:\Programs type paths,, but in \Users\[user]\AppData.

I get the vague feeling that it's a 'cloud thing,' (not that I know what a cloud actually is. ;)) because that's what F360 is clearly touted as. Or at least -related- to it on some security or user interface level. I haven't seen mention of 'cloud' related to GH but GH and F360 definitely have some very similar 'features.' (Or FORCED INTRUSIONS depending on how you think about them ;)).

Examples. Identical user data resides essentially simultaneously on the local device, and on the server. The user can operate on data whether connected or not. Both systems deeply integrate data sharing -- it's a fundamental feature.

One DRASTIC difference is that one needn't use the GH GUI to use the GH server system. I'm actually very new to GH but have read that the GUI is pretty recent. OTOH with F360 the server residence absolutely-purely shoved down the user's throat -- it  cannot operate in any mode other than as supplied.

Related to that, in GH one can directly edit text data on the server, and to have that data on the local device thee user must explicitly tell the data to move around. In F360 that 'online editing' is simply not possible.

Well, obviously I don't have anything to really contribute to solving the problem. It is what it is. But maybe it'll help some understand -what's- happening even if we don't know exactly -why-, or -how- to fix it.

Also an FTR, I don't know what a 'pin' is, what it does, or how to use it. :) I've seen the word around forever but even the microsoft pages only explain -how- to use it, (something related to taskbar,) not why it's there or what it does.

THANKS BILL!!!!!!!!! I'm sure you're a really nice guy but you have noooooooooooooo clue how people use their PCs in the Real World where -we- live. ;)
